FhotoFithFriends/src/main/resources/templates/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fhoto Fith Friends</title>
+    <link rel="stylesheet" href="css/bootstrap.min.css" />
+    <link rel="stylesheet" href="css/styles.css" />
+</head>
+
+<body>
+    <div class="container">
+        <h1 id="title">Fhoto Fith Friends</h1>
+        <div id="home-screen">
+        <p id="stuff">Behold, My Stuff!</p>
+        </div>
+    </div>
+    <script src="js/index.js"></script>
```

Comment:
Created resources/static/js/ and index.js for future scripting and API possibilities